### PR TITLE
Uptimekuma

### PR DESCRIPTION
**Pull request by ivanonpc-22**
Working UptimeKuma template. Let me know if any changes need to be made.

### CODE REVIEW

**Comment by ivanonpc-22** [comment]
whoops let me fix that real quick

File: templates/uptimekuma/index.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        },
+        proxy: {
+          port: 3001,
+          secure: true,
+        },
+        domains: [{ name: domain }],
+      mounts: [
+          {
+            type: "volume",
+            name: "uptimekuma",
```
Comment (deiucanta) [inline templates/uptimekuma/index.ts:59]: Name it `data`. Each volume is scoped by the project so you don't have to worry about name conflicts.

**Review by deiucanta — CHANGES_REQUESTED** [review]

**Comment by deiucanta** [comment]
Thanks @ivanonpc-22 for your contribution! I will include this in the next release :)

**Comment by ivanonpc-22** [comment]
I will keep them coming! Have a great day.

**Pull request merged**